src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/BookingResourceIntTest.kt

```diff
@@ -189,21 +188,21 @@ class BookingResourceIntTest : ResourceTest() {
     }
 
     @Test
-    @Disabled("this test fails - code update needed")
-    fun `should return 403 if does not have override role`() {
+    fun `should return empty list if client does not have override role`() {
       webTestClient.get().uri("/api/bookings/v2?prisonId=BXI")
         .headers(setClientAuthorisation(listOf()))
         .exchange()
-        .expectStatus().isForbidden
+        .expectStatus().isOk
+        .expectBody().jsonPath("totalElements").isEqualTo(0)
```

Comment:
Have agreed with Andy M that due to the sql filtering the data - this was acceptable